### PR TITLE
option for user input

### PR DESCRIPTION
**Pull request by drummerwolli**

### CODE REVIEW

File: tunnels.py [hunk 1]
```diff
@@ -44,7 +45,7 @@ def cli(stack_name, port, jump_host, region):
     click.secho('Endpoints: {}'.format(','.join(endpoints)), bold=True, fg='blue')
 
     click.secho('Starting SSH tunnels..', bold=True)
-    subprocess.call(['ssh'] + opts + [jump_host, 'while true; do echo -n .; sleep 60; done'])
+    subprocess.call(['ssh'] + opts + [user + '@' + jump_host, 'while true; do echo -n .; sleep 60; done'])
```
Comment (hjacobs) [inline tunnels.py]: this will not work if no user is specified

File: tunnels.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 @click.argument('port', type=int)
 @click.argument('jump_host')
 @click.option('--region')
+@click.option('--user')
 def cli(stack_name, port, jump_host, region):
```
Comment (hjacobs) [inline tunnels.py]: the "user" argument is missing here

File: tunnels.py [hunk 3]
```diff
@@ -22,6 +22,7 @@
 @click.argument('port', type=int)
 @click.argument('jump_host')
 @click.option('--region')
+@click.option('--user')
```
Comment (hjacobs) [inline tunnels.py]: add the short option "-U" too

**Comment by drummerwolli** [comment]
see https://github.com/drummerwolli/ssh-tunnels/commit/b30ba0e7bd8d6566c134a95869e329aaa44c7263

**Comment by hjacobs** [comment]
@drummerwolli please consider using "flake8" always on your *.py files (code formatting and simple static analysis)

**Pull request merged**